package.json

```diff
@@ -12,7 +12,6 @@
     "docker:build": "npm run docker:build:base && npm run docker:build:network && npm run docker:build:monitor && npm run docker:build:ingestor && npm run docker:build:amplify",
     "docker:compose": "docker compose --file docker/colony-cdapp-dev-env-orchestration.yaml",
     "dev": "npm run docker:build && npm run docker:compose up -- --force-recreate -V",
-    "lint": "eslint --fix 'src/**/*.{ts,tsx}'",
```

Comment:
No need for own command, right? (I'm assuming no one uses this). I propose we use staged-lint as the single source of truth for linting commands.

Reply:
You may want to run linting for the entire repo (probably something we'd want to do especially when trying to clean up all the errors). `lint-staged` only runs the linting for staged changes. I'd leave it there

Reply:
I use this command in the Dapp. I think it could be useful in the beginning to catch stuff that may not be worked on, and therefore, no staging changes. After we are sure that we are lint-error-free (as we are currently in the Dapp for example), we could remove it